src/mixins/filing-template-mixin.ts

```diff
@@ -702,8 +730,9 @@ export default class FilingTemplateMixin extends DateMixin {
 
     // store Business Information
     this.setBusinessInformation({
+      ...entitySnapshot.businessInfo,
       ...filing.business,
-      ...filing.alteration?.business
+      ...filing.alteration.business
```

Comment:
Would it be better if this was `...filing.alteration.business?`

Reply:
I was going to say no since we're not directly de-referencing any properties from `filing.alteration.business`, but we sort of are with the destructuring syntax.

I was also going to say that it should be safe to add the trailing `?`, but that actually results in a syntax error:

![image](https://user-images.githubusercontent.com/10002889/230175368-0597228c-e35e-40da-ac7e-7538e5e870b7.png)

The following works as expected:

![image](https://user-images.githubusercontent.com/10002889/230175719-62057f34-eb64-47a5-a963-ad9b4ba86864.png)

Therefore I'm thinking we should leave this alone. Do you have further insight on this (eg, a failing example)?

Reply:
Hmm, there shouldn't be a failing example but I was thinking that maybe for whatever reason we were unable to access the business property. Also, since we use this optional chaining on line 729. No worries, let's keep it as is. Going to mark this conversation as resolved!

Reply:
On line 729, we explicitly de-reference a property from an object that might be null or undefined.

Your point makes sense: I'm a bit surprised that JS can destructure an undefined object, but my test above shows it's OK.

Reply:
I found this: https://stackoverflow.com/questions/46957194/javascript-es6-spread-operator-on-undefined

Now we can sleep soundly tonight.

Reply:
Ohh, got it got it. Thank you for the reference Sev.